modules/albo-installation.adoc

```diff
@@ -0,0 +1,282 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: PROCEDURE
+[id="aws-load-balancer-operator-installation_{context}"]
+= Installing the AWS Load Balancer Operator
+
+After setting up your environment with your cluster, you can install the AWS Load Balancer Operator using the CLI.
+
+.Procedure
+. Create an AWS IAM policy for the AWS Load Balancer Controller:
++
+[NOTE]
+====
+The policy is sourced from link:https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.4/docs/install/iam_policy.json[the upstream AWS Load Balancer Controller policy] plus permission to create tags on subnets. This is required by the Operator to function.
```

Comment:
The "plus" is unclear here - do we mean that we have used the policy from kubernetes-sigs as a base, and the only addition made is the permission to create tags on subnets?

Reply:
I didn't write this, but since the linked policy has:
![image](https://github.com/user-attachments/assets/9c6eb7d6-3635-4db8-9539-b975a2dee8d3)

I revised it to:

```suggestion
You can find the AWS IAM policy from link:https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.4/docs/install/iam_policy.json[the upstream AWS Load Balancer Controller policy]. This policy includes all of the permissions you needed by the Operator to function.
```